frontend/src/components/reviewCard/reviewCard.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { StarIcon } from "@heroicons/react/20/solid";
+import clsx from "clsx";
+import { format } from 'date-fns';
+
+// 
+
+function ReviewCard(props) {
```

Comment:
can we de-structure review in the in the function params and change this to an arrow function:

`const ReviewCard = ({review}) => {`